### PR TITLE
Add FAQ for paused field

### PR DESCRIPTION
**Pull request by hardikdr**
**What this PR does / why we need it**: Added an faq for a `paused` field in machine-deployment.

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```improvement operator

```

### CODE REVIEW

**Review by prashanth26 — APPROVED** [review]
/lgtm

File: docs/FAQ.md [hunk 1]
```diff
@@ -110,6 +111,19 @@ Each `machine` object has an annotation `machinepriority.machine.sapcloud.io` se
 
 A machine can be force deleted by adding the label `force-deletion: "True"` on the `machine` object if it's already being deleted. During force deletion, MCM skips the drain function and simply triggers the deletion of the machine. This label should be used with caution as it can violate the PDBs for pods running on the machine.
 
+### How to pause the ongoing rolling-update of the machinedeployment?
+
+An ongoing rolling-update of the machine-deployment can be paused by using `Spec.Paused` field. See the example below:
```
Comment (dguendisch) [inline docs/FAQ.md:116]:
```suggestion
An ongoing rolling-update of the machine-deployment can be paused by using `spec.paused` field. See the example below:
```

**Review by dguendisch — COMMENTED** [review]
only one nit

**Review by dguendisch — COMMENTED** [review]
thanks :)

**Review by prashanth26 — APPROVED** [review]
/lgtm

**Pull request merged**